benches/throughput.rs

```diff
@@ -47,7 +47,7 @@ fn throughput(c: &mut Criterion) {
             |b, &num_messages| {
                 b.to_async(&runtime).iter(|| async {
                     for _ in 0..num_messages - 1 {
-                        address.do_send(IncrementZst).unwrap();
+                        address.send(IncrementZst).await.unwrap();
```

Comment:
I think this should probably be
```rust
let _ = address.send(IncrementZst).recv_async().await;
```
Otherwise, the benchmark would be the same no matter the batch size of `num_messages`, would it not?

Reply:
I didn't pay too much attention to this to be honest but yes, this makes total sense!